rinja/Cargo.toml

```diff
@@ -26,11 +26,20 @@ harness = false
 [dependencies]
 rinja_derive = { version = "=0.3.5", path = "../rinja_derive" }
 
+itoa = "1.0.11"
+
+# needed by feature "urlencode"
 percent-encoding = { version = "2.1.0", optional = true, default-features = false }
+
+# needed by feature "serde_json"
 serde = { version = "1.0", optional = true, default-features = false }
-serde_json = { version = "1.0", optional = true, default-features = false, features = [] }
+serde_json = { version = "1.0", optional = true, default-features = false }
 
-itoa = "1.0.11"
+# needed by feature "dynamic"
+linkme = { version = "0.3.31", optional = true }
+notify = { version = "8.0.0", optional = true }
+parking_lot = { version = "0.12.3", optional = true, features = ["arc_lock", "send_guard"] }
+tokio = { version = "1.43.0", optional = true, features = ["macros", "io-std", "io-util", "process", "rt", "sync", "time"] }
```

Comment:
* `linkme` and `notify` are absolutely needed.
* I tried to implement the PR without an `async` runtime, but well, tokio is just too well implemented to invent the wheel twice.
* `parking_lot` could maybe be replace with an `std::sync::Barrier` of some sort. It's far from trivial, though.